### PR TITLE
Changes for Nodo version

### PR DESCRIPTION
**Pull request by tjfsteele**

### CODE REVIEW

File: software/wordclock/src/ClockFace.cpp [hunk 1]
```diff
@@ -274,6 +291,16 @@ bool FrenchClockFace::stateForTime(int hour, int minute, int second, bool show_a
 
   switch (leftover)
   {
+#ifdef NODO
+  case 4:
+    _state[113] = true;
```
Comment (DaveDavenport) [inline software/wordclock/src/ClockFace.cpp:296]: Why not do this in the mapminute() method, then the change only required at single point?

Comment (tjfsteele) [reply software/wordclock/src/ClockFace.cpp:296]: You're right, I'll do that.

**Comment by tjfsteele** [comment]
Changed nodo.h to build the standard version by default

File: software/wordclock/wordclock.ino [hunk 2]
```diff
@@ -1,30 +1,32 @@
...
 #include "src/Display.h"
 #include "src/ClockFace.h"
 #include "src/Iot.h"
-
+#include "src/nodo.h" // Nodo stuff
 RTC_DS3231 rtc;
-EnglishClockFace clockFace(ClockFace::LightSensorPosition::Bottom);
-// FrenchClockFace clockFace(ClockFace::LightSensorPosition::Bottom);
+
+EnglishClockFace clockFace(ClockFace::LightSensorPosition::Top);
```
Comment (t0mg) [inline software/wordclock/wordclock.ino:10]: Please leave the default bottom here too.

**Review by t0mg — CHANGES_REQUESTED** [review]
Please also leave the default sensor position so the update is transparent. 

I wonder it we should properly define corner LEDs dor NODO with sensor at the bottom too.

**Comment by tjfsteele** [comment]
I've changed that so the default build will be with the sensor position as Bottom.

**Pull request merged**